README.md

```diff
@@ -7,7 +7,7 @@ Current stable release versions are
 
     API version: 1.0.0
     SDK version: 0.0.12
-    CLI version: 0.0.12
+    CLI version: 0.0.14
```

Comment:
@nazeelashraf  Any reason for not using the version number 0.0.13?

Reply:
13 was already mentioned in `setup.py`, so I went with 14 to avoid any confusion.